_scripts/deploy.sh

```diff
@@ -10,3 +10,9 @@ docker login -e="$DOCKER_EMAIL" -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
 DEIS_REGISTRY='' make -C .. docker-build docker-push
 docker login -e="$QUAY_EMAIL" -u="$QUAY_USERNAME" -p="$QUAY_PASSWORD" quay.io
 DEIS_REGISTRY=quay.io/ make -C .. docker-build docker-push
+
+# download deis CLI & deploy to deis
+curl -sSL http://deis.io/deis-cli/install-v2.sh | bash
+mv ./deis /usr/local/bin/deis
+deis login --username=$DEIS_USERNAME --password=$DEIS_PASSWORD
```

Comment:
Where are `DEIS_USERNAME` and `DEIS_PASSWORD` set? Assume we're expecting them to be set in CI environ -- do we need to set these to some defaults if both/either are empty?